### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by zalexchen**
Updating Main with dev in preparation for integration with travis-ci

### CODE REVIEW

File: app/src/main/java/com/example/warriorsocial/ui/organizations/OrganizationsFragment.java [hunk 1]
```diff
@@ -92,10 +91,12 @@ protected void onBindViewHolder(OrganizationsViewHolder viewHolder, int position
                 System.out.println("inside onBindViewHolder in OrganizationsFragment");
                 final DatabaseReference StudentOrganizationRef = getRef(position);
                 // For Testing
+                /*
                 System.out.println("StudentOrganizationRef KEY: " + StudentOrganizationRef.getKey());
                 System.out.println("StudentOrganizationRef PARENT: " + StudentOrganizationRef.getParent());
                 System.out.println("StudentOrganizationRef REF: " + StudentOrganizationRef.getRef());
                 System.out.println("StudentOrganizationRef ROOT: " + StudentOrganizationRef.getRoot());
+                */
```
Comment (zalexchen) [inline app/src/main/java/com/example/warriorsocial/ui/organizations/OrganizationsFragment.java:99]: Test Code was cleaned up but there is still many "print out statements" lingering.

File: app/src/main/res/layout/activity_orgs.xml [hunk 2]
```diff
@@ -10,6 +10,25 @@
         android:layout_height="match_parent"
         tools:context=".ui.organizations.OrganizationProfile">
 
+        <RelativeLayout
```
Comment (Sher1nShaj1) [inline app/src/main/res/layout/activity_orgs.xml:13]: Recycler view for the posts looks good. Nice job on the tab layout functionality

File: app/src/test/java/com/example/warriorsocial/ui/login/RegistrationCheckerTest.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class RegistrationCheckerTest {
+
+    @Test
+    public void checkIfUserLoggedIn() {
+        fail();
```
Comment (ChloeMJM) [inline app/src/test/java/com/example/warriorsocial/ui/login/RegistrationCheckerTest.java:11]: Is this complete? Looks like a few tests are unfinished.

File: app/src/main/res/layout/student_organization_card.xml [hunk 4]
```diff
@@ -1,40 +1,63 @@
...
+    android:layout_margin="10dp"
+    android:orientation="vertical">
 
-    <androidx.constraintlayout.widget.ConstraintLayout
+    <androidx.cardview.widget.CardView
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
-
-
-        <TextView
```
Comment (rahulsheth1) [inline app/src/main/res/layout/student_organization_card.xml:20]: Great job cleaning up the files and removing unused code.

File: app/src/test/java/com/example/warriorsocial/ui/login/RegistrationCheckerTest.java (hunk 3 above)
Comment (zalexchen) [reply app/src/test/java/com/example/warriorsocial/ui/login/RegistrationCheckerTest.java:11]: New Update for dev fixes this. This was an incomplete version, opened a pull request from AlexChen_UnitTest (tested myself runs unit tests fine) to dev!

**Pull request merged**